src/Sylius/Component/Core/spec/OrderProcessing/OrderShipmentProcessorSpec.php

```diff
@@ -273,4 +314,47 @@ function it_does_nothing_if_the_order_cannot_be_processed(OrderInterface $order)
 
         $this->process($order);
     }
+
+    function it_does_not_add_item_units_that_do_not_require_shipping(
+        FactoryInterface $shipmentFactory,
+        OrderInterface $order,
+        ShipmentInterface $shipment,
+        OrderItemUnitInterface $shippableUnit,
+        OrderItemUnitInterface $nonShippableUnit,
+        ProductVariantInterface $shippableVariant,
+        ProductVariantInterface $nonShippableVariant,
+        DefaultShippingMethodResolverInterface $defaultShippingMethodResolver,
+        ShippingMethodInterface $shippingMethod,
```

Comment:
In this class, shippingMethodResolver appears to be a first argument in multiple methods -- let's do the same.

P.S. I've moved ShipmentInteface after Order* arguments similar to other places as well.

```suggestion
        DefaultShippingMethodResolverInterface $defaultShippingMethodResolver,
        FactoryInterface $shipmentFactory,
        OrderInterface $order,
        OrderItemUnitInterface $shippableUnit,
        OrderItemUnitInterface $nonShippableUnit,
        ShipmentInterface $shipment,
        ProductVariantInterface $shippableVariant,
        ProductVariantInterface $nonShippableVariant,
        ShippingMethodInterface $shippingMethod,
```